devices/sonoff_basic.rst

```diff
@@ -54,7 +54,8 @@ exposes all of the basic functions.
     esphome:
       name: <NAME_OF_NODE>
       platform: ESP8266
-      board: esp01_1m
+      board: esp8285
+      arduino_version: 2.4.2
```

Comment:
I wonder if this Arduino downgrade is still needed. This is based on information from 2019...

Reply:
I have a serial console connected to the one on my desk.

Will test without that arduino version line this afternoon and let you know how it goes..

Reply:
Did not go well.

Never reconnected to wifi and lucky for me I had the serial console so I could easily flash back the 2.4.2 version.

These devices are very finicky but using that declared arduino_version does seem to make a difference for the better.

Reply:
Thanks! 👍